packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -278,10 +278,11 @@ contract VotingV2 is
 
         // Traverse all requests from the last considered request. For each request see if the voter voted correctly or
         // not. Based on the outcome, attribute the associated slash to the voter.
-        for (uint256 i = voterStake.lastRequestIndexConsidered; i < priceRequestIds.length; i++) {
-            PriceRequest storage priceRequest = priceRequests[priceRequestIds[i].requestId];
+        Request[] memory priceRequestIdsMemo = priceRequestIds;
```

Comment:
```
  VotingV2 gas usage
Gas used variation for "Simple vote resolution": -30
New gas used: 589870
    ✓ Simple vote resolution
Gas used variation for "Batches multiple commits into one": -8265
New gas used: 1047746
    ✓ Batches multiple commits into one
Gas used variation for "Batch reveal multiple commits": -614
New gas used: 374572
    ✓ Batch reveal multiple commits
Gas used variation for "Voter doesn't vote in 100 rounds": -48774
New gas used: 7926964
    ✓ Voter doesn't vote in 100 rounds
```

Reply:
interesting. there might be other changes similar to this that we can make through how we do storage/memory. shall we call this variable `priceRequestIds` rather?